crates/rune/src/runtime/unit.rs

```diff
@@ -79,18 +108,25 @@ impl<S> Unit<S> {
         constants: HashMap<Hash, ConstValue>,
     ) -> Self {
         Self {
-            storage,
-            functions,
-            static_strings,
-            static_bytes,
-            static_object_keys,
-            rtti,
-            variant_rtti,
+            data: UnitData {
+                storage,
+                functions,
+                static_strings,
+                static_bytes,
+                static_object_keys,
+                rtti,
+                variant_rtti,
+                constants,
+            },
             debug,
-            constants,
         }
     }
 
+    /// Access unit data.
+    pub fn unit_data(&self) -> &UnitData<S> {
+        &self.data
+    }
+
     /// Access debug information for the given location if it is available.
```

Comment:
I wasn't entirely sure what

> for the given location

meant in this context, so I left it off `Self::unit_data`'s documentation.

If `DebugInfo` is contextual on the state of the unit, then perhaps it could just be removed from serialisation entirely with `#[serde(skip)]`

Reply:
I think it's just a holdover comment from when this accessor took an instruction pointer. So it shouldn't be more contextual than anything else in the unit.